src/test/resources/application-unit-test.yml

```diff
@@ -0,0 +1,4 @@
+workspace:
```

Comment:
can you just leave this file out altogether if you want all defaults?

Reply:
Not sure what you mean. The default configuration - which is the production config - is in CrlConfiguration.java and says use CRL and do not use Janitor.

This file is included by the Spring framework when the active profile is "unit-test" and it sets the config properly for unit tests. The BaseUnitTest.java file sets the active profile.

Reply:
sounds good